### PR TITLE
packaging: fix rhel8 package build

### PR DESCRIPTION
**Pull request by safchain**
skip skydive-go-fmt
skip skydive-functional-tests-backend-elasticsearch
skip skydive-functional-tests-backend-orientdb
skip skydive-scale-tests
skip skydive-cdd-overview-tests
skip skydive-unit-tests
skip skydive-compile-tests
skip skydive-k8s-tests
skip skydive-ppc64le-tests
skip skydive-functional-tests-ebpf
skip skydive-functional-hw-tests

### CODE REVIEW

**Comment by lebauce** [comment]
run skydive-packaging-tests

File: contrib/packaging/rpm/skydive.spec [hunk 1]
```diff
@@ -115,7 +115,7 @@ This package installs and sets up the SELinux policy security module for Skydive
 
 %build
 export GOPATH=%{_builddir}/skydive-%{fullver}
-make compile BUILD_CMD=go VERSION=%{fullver} %{with_features}
+make compile BUILD_CMD=go VERSION=%{fullver} LDFLAGS="-compressdwarf=false" %{with_features}
```
Comment (lebauce) [inline contrib/packaging/rpm/skydive.spec]: This should be done only for f27+ and rhel 8+

**Comment by lebauce** [comment]
skip skydive-go-fmt
skip skydive-functional-tests-backend-elasticsearch
skip skydive-functional-tests-backend-orientdb
skip skydive-scale-tests
skip skydive-cdd-overview-tests
skip skydive-unit-tests
skip skydive-compile-tests
skip skydive-k8s-tests
skip skydive-ppc64le-tests
skip skydive-functional-tests-ebpf
skip skydive-functional-hw-tests

**Comment by lebauce** [comment]
run run skydive-packaging-tests

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by lebauce** [comment]
run skydive-packaging-tests

**Comment by lebauce** [comment]
run skydive-packaging-tests

**Comment by lebauce** [comment]
run skydive-packaging-tests

**Comment by lebauce** [comment]
run skydive-packaging-tests

**Comment by lebauce** [comment]
run skydive-packaging-tests

**Comment by lebauce** [comment]
run skydive-packaging-tests

**Comment by lebauce** [comment]
run skydive-packaging-tests

**Comment by lebauce** [comment]
run skydive-packaging-tests

**Comment by lebauce** [comment]
run skydive-packaging-tests

**Comment by lebauce** [comment]
run skydive-packaging-tests

**Pull request merged**